--- a/telemetrix-1.46.dist-info/RECORD
+++ b/telemetrix-1.46.dist-info/RECORD
@@ -6,7 +6,6 @@
 archive/stepper_info_play.py,sha256=HswxocunurTP5rMjsMkxNbeWf0I_SWhnNDQgIxZy_IU,815
 archive/tcp_receiver.py,sha256=FIt8pB4nviZWyOim8rhdJ9Pu-tDz7T2uPigWqiJrThg,263
 archive/telemetrix-11-12.py,sha256=6eZg5i2ez_Lc_Fi2rAC0o4yTGjhPhmaw8VDF2nbpgV4,37991
-archive/telemetrix_10_17_10:14.py,sha256=x7IJs5i4cy1MMK67Bf_Pw_fhsUCJRWT9HEAtzsghhmo,35207
 archive/telemetrix_aio_socket.py,sha256=bssMEtU2XFMBb6cblZZanA1EJcKEoAS9U7_IOIMgmjw,2680
 archive/udp2.py,sha256=Hzgk9wbA_yq5Hu-UWLFQm9fpy93lAuy9inLiLJ-JgIM,568
 arduino_json_evaluation/deserialize/__init__.py,sha256=47DEQpj8HBSa-_TImW-5JCeuQeRkm5NMpJWZG3hSuFU,0
